src/goto-symex/symex_target_equation.cpp

```diff
@@ -645,12 +646,15 @@ Function: symex_target_equationt::convert_assignments
 \*******************************************************************/
 
 void symex_target_equationt::convert_assignments(
-  decision_proceduret &decision_procedure) const
+  decision_proceduret &decision_procedure)
 {
```

Comment:
I have to update step.converted to remember which steps are already in the solver.